SoftwareDesignDocument..txt

```diff
@@ -69,3 +69,13 @@ Req 1.2.1
         The texture for the block will be added in the textures folder, in the texture.png file.
 Code:        
 	TANK_SHELL = 24
+Req 1.3, 1.3.1
+	When the user types "/play tank minigame" into the command, spawn the user into an arena at a fixed location.
```

Comment:
I think we need a something that will redirect it to the function from the console command when using "/play tank minigame"

Reply:
I would add a pseudo code function to declare what you mean prior to going to implementation.